src/config.ts

```diff
@@ -1,15 +1,39 @@
 import { PublicKey } from "@solana/web3.js";
 import { DIPDeposit } from "./common";
 
-export const IS_DEV: boolean = true;
+export const IS_DEV = process.env.DEV == "false" ? false : true;
+export const API_URL = process.env.RPC;
+export const sol_vars = process.env.SOL.split(",");
+export const btc_vars = process.env.BTC.split(",");
+export const eth_vars = process.env.ETH.split(",");
+export const mngo_vars = process.env.MNGO.split(",");
+
 export const productStatus = new Map<string, boolean> ([
-  ['BTC', false], ['ETH', false], ['SOL', true], ['MNGO', false]
+  ['BTC', btc_vars[0] == "ON" ? true : false], 
+  ['ETH', eth_vars[0] == "ON" ? true : false], 
+  ['SOL', sol_vars[0] == "ON" ? true : false], 
+  ['MNGO', mngo_vars[0] == "ON" ? true : false]
+]);
+// Adjust delta hedges for loans, negative values allow positive spot balances in mango
+// CAUTION! Turn off scalper, send funds to mango & update value before running!
```

Comment:
better than a warning here in the code would be to log the delta offset at the start of a scalper run, and if we think it is necessary, add a roadblock of some kind.

Reply:
separated offset delta from spot delta and logging it now

Reply:
![image](https://user-images.githubusercontent.com/107803907/205201165-6222629a-ee2b-42ef-89bb-8dd99a0e7c1a.png)